dipy/tracking/local_tracking.py

```diff
@@ -89,6 +104,19 @@ def __init__(self, direction_getter, stopping_criterion, seeds, affine,
             raise ValueError("maxlen must be greater than or equal to minlen")
         if not isinstance(seeds, Iterable):
             raise ValueError("seeds should be (N,3) array.")
+        if (initial_directions is not None and
+                seeds.shape[0] != initial_directions.shape[0]):
+            raise ValueError("initial_directions and seeds must have the "
+                             + "same shape[0].")
+        if (initial_directions is None and unidirectional and
+                self.randomize_forward_direction is False):
```

Comment:
I will avoid to use `is False`.  instead can you use `not self.randomize_forward_direction`

Reply:
yes, thanks. fixed.